src/modules/Common/components/Card.tsx

```diff
@@ -49,14 +59,20 @@ const Card: React.FC<CardProps> = ({
         {children && <div className={s.card_children}>{children}</div>}
       </div>
       {author && (
-        <div className={s.card_author}>
-          <div className={s.card_author_icon}>
-            <FiUser color="#999999" />
-          </div>
+        <div
+          className={classNames(s.card_author, {
+            [s.card_author__isCenter]: authorCenter === true,
```

Comment:
`authorCenter` is already a boolean `[s.card_author__isCenter]: authorCenter,`

Reply:
https://github.com/ambanum/OpenTermsArchive.org/pull/161/commits/dbcd2ac2735d150a6e5718f10bb4ebba22c7bddd